app/move/controllers/create/assessment.js

```diff
@@ -1,11 +1,11 @@
 const { flatten, values } = require('lodash')
 
-const FormController = require('./new.form')
-const fieldHelpers = require('../../../common/helpers/field')
-const referenceDataService = require('../../../common/services/reference-data')
-const referenceDataHelpers = require('../../../common/helpers/reference-data')
+const CreateBaseController = require('./base')
```

Comment:
~do we need `base` in this name? Would `CreateController` do?~

Reading through this I can see why you have used `base`